tests/php/Extension/FluentVersionedExtensionTest.php

```diff
@@ -206,4 +211,164 @@ public function testStagesDifferInLocale(): void
+            $this->assertTrue($localisedRecord->isLatestVersion());
+            $this->assertTrue($localisedRecord->isLiveVersion());
+            $this->assertFalse($localisedRecord->isModifiedOnDraft());
+            $this->assertFalse($localisedRecord->isOnLiveOnly());
+        });
+
+        // Check still only true for draft stuff in *alternate* locale
+        FluentState::singleton()->withState(function (FluentState $newState) use ($recordID) {
+            $newState->setLocale('en_US');
+            $localisedRecord = TestVersionedModel::get()->byID($recordID);
+            $this->assertTrue($localisedRecord->isOnDraft());
+            $this->assertFalse($localisedRecord->isPublished());
+            $this->assertTrue($localisedRecord->isLatestDraftVersion());
+            $this->assertTrue($localisedRecord->isLatestVersion());
+            $this->assertFalse($localisedRecord->isLiveVersion());
+            $this->assertTrue($localisedRecord->isModifiedOnDraft());
+            $this->assertFalse($localisedRecord->isOnLiveOnly());
+
+            // Then publish it in this locale
+            $localisedRecord->publishSingle();
+        });
+
+        // Update, save, and publish the record in first locale
+        FluentState::singleton()->withState(function (FluentState $newState) use ($recordID) {
+            $newState->setLocale('en_NZ');
+            $localisedRecord = TestVersionedModel::get()->byID($recordID);
+            $localisedRecord->Title = 'New value';
+            $localisedRecord->write();
+            $localisedRecord->publishSingle();
+
+            // True for anything about being published and "on draft" (i.e. not deleted from draft)
+            $this->assertTrue($localisedRecord->isOnDraft());
+            $this->assertTrue($localisedRecord->isPublished());
+            $this->assertTrue($localisedRecord->isLatestDraftVersion());
+            $this->assertTrue($localisedRecord->isLatestVersion());
+            $this->assertTrue($localisedRecord->isLiveVersion());
+            $this->assertFalse($localisedRecord->isModifiedOnDraft());
+            $this->assertFalse($localisedRecord->isOnLiveOnly());
+        });
+
+        // Check *alternate* locale still thinks it's published - and on the latest published version
+        FluentState::singleton()->withState(function (FluentState $newState) use ($recordID) {
+            $newState->setLocale('en_US');
+            $localisedRecord = TestVersionedModel::get()->byID($recordID);
+            $this->assertTrue($localisedRecord->isOnDraft());
+            $this->assertTrue($localisedRecord->isPublished());
+            $this->assertTrue($localisedRecord->isLatestDraftVersion());
+            $this->assertTrue($localisedRecord->isLatestVersion());
+            $this->assertTrue($localisedRecord->isLiveVersion());
+            $this->assertFalse($localisedRecord->isModifiedOnDraft());
+            $this->assertFalse($localisedRecord->isOnLiveOnly());
+        });
+    }
```

Comment:
Can you please add some test cases for the UK locale? This one inherits content from another locale.